### PR TITLE
INT-B-21042

### PR DESCRIPTION
**Pull request by loganwc**
## Summary

Allows Prime to create shipments with second and third pickup/destination addresses.

## Testing
1. Login as Prime
2. Open existing move
3. Click Create Shipment
4. Enter 2nd/3rd addresses
5. Save
6. Check DB for 3rd addresses

### CODE REVIEW

**Comment by pambecker** [comment]
> Boat haul away and tow away saved, but did not retain second/third address values. Mobile home shipment failed with second/third addresses.
> 
> Are those within the scope of this work?

As discussed in team sync today, Boat and Mobile home will be future work and not part of this story.

**Comment by pambecker** [comment]
Found an issue, when I say yes to 2nd addr and 3rd addr and enter information in the fields.  Then go back and select No for 2nd addr (before saving the new shipment), it's not hiding the 3rd addr on the screen, and not nulling out the 2nd and 3rd addresses.  So when I go to save, it still saves the 2nd and 3rd addr when it shouldn't be.

File: src/pages/PrimeUI/Shipment/PrimeUIShipmentCreateForm.jsx [hunk 1]
```diff
@@ -535,7 +535,7 @@ const PrimeUIShipmentCreateForm = () => {
                     </FormGroup>
                   </>
                 )}
-                {hasTertiaryDestinationAddress === 'true' && (
+                {hasTertiaryPickupAddress === 'true' && hasSecondaryPickupAddress === 'true' && (
```
Comment (pambecker) [inline src/pages/PrimeUI/Shipment/PrimeUIShipmentCreateForm.jsx:545]: I think maybe a copy and paste error here?  we want this to look at destinationAddress

Comment (loganwc) [reply src/pages/PrimeUI/Shipment/PrimeUIShipmentCreateForm.jsx]: fixed

**Review by pambecker — APPROVED** [review]

**Review by paulstonebraker — APPROVED** [review]

**Pull request merged**